modules/i18n/src/main/Translation.scala

```diff
@@ -19,7 +19,7 @@ private class Literal(message: String, escapedOption: Option[String]) extends Tr
     else Html(escaped.format(args.map(_.body): _*))
 }
 
-private class Plurals(messages: Map[I18nQuantity, String]) extends Translation {
+private case class Plurals(messages: Map[I18nQuantity, String]) extends Translation {
```

Comment:
`case class` add some overhead. Is this overhead linear to the number of instanciated objects? I don't know. But if it is the case, then it's a concern, for we have a lot of `Literal` and `Plurals` instances in the heap.